src/api/app/models/notification.rb

```diff
@@ -40,6 +42,13 @@ def unread?
   def unread_date
     last_seen_at || created_at
   end
+
+  private
+
+  def track_notification_creation
+    RabbitmqBus.send_to_bus('metrics',
+                            "notification.create,notifiable_type=#{notifiable_type},notifiable_id=#{notifiable_id},web=#{web},rss=#{rss} value=1")
```

Comment:
`notifiable_id` is not really interesting and has a very high cardinality. This is to be avoided as it will lead to a slow index in the TSDB.

Reply:
You are right! I planned to use it to detect requests that had a lot of activity/notifications. But it's not really so important.

I've removed that tag. Please have another look.

Reply:
@hennevogel this is ready for another review